### DISCUSSION TITLE
I can't reverify my student status using "dated receipt from bursar"

### DISCUSSION THREAD

**Discussion by lazy-geek** [community/community#50308] [GitHub Education]
When i try to reverify my student status by using "dated receipt from bursar" as proof, it keeps getting rejected & says to use "dated school id"
![Screenshot_2023-03-17-16-00-34-574_com android chrome](https://user-images.githubusercontent.com/38218691/225881739-e1e4b1db-7b51-4b55-a76b-2b85ecc057d2.jpg)

I tried to use my college id but as my college id is 2 sided and 1 side with photo & my name & other side with the valid date written on it, i created an image collage & uploaded it as single image but i still get the same error.

**Comment by Mohamed-SayedAlAhl** [comment]
It sounds like you are having trouble re-verifying your student status using a dated receipt from the bursar. The system keeps rejecting it and asking for a dated school ID instead. 

One thing you could try is to reach out to the support team of the platform or service you are trying to re-verify your student status for. They may be able to provide more information on why your receipt is not being accepted and offer suggestions on what other types of documentation may be acceptable. 

Another option is to reach out to your school's bursar office or student services center to see if they can provide a different type of verification document that may be accepted. 

Lastly, it's possible that the issue may be with the image collage you created. Double-check that the image is clear and legible, and make sure it meets the platform's requirements for image size and file type. If possible, try uploading each side of your ID separately to see if that helps. 

I hope this helps you resolve your issue and successfully re-verify your student status!

**Reply by lazy-geek** [reply]
I can't write to support about this because of these changes: https://github.com/orgs/community/discussions/49366

It doesn't let me use anything other that school id, **thats** the issue, so i don't know how getting a different proof may help.
And as for school id , I can't upload 2 photos individually, so i **have** to make a collage.

**Comment by samus-aran** [comment]
Hi @lazy-geek 👋 

We're happy you're here, lets see how we can assist.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

Do you have an academic email. As it's the preferred evidence we would suggest you reach out to your institution to see if you are eligible.

We do accept Student ID cards without dates so please continue to use this as evidence. We can also suggest you take a photo of multiple pieces of evidence side by side. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either.

Hope this helps 👍

**Reply by lazy-geek** [reply]
Thanks for the reply,I don't have academic email but i will try to take higher resolution photos of my student id and try to submit that.

**Comment by onyedikachi23** [comment]
**Please don't do this if you feel your privacy might be compromised**:

Can you upload the snaps of the receipt?

**Comment by lazy-geek** [comment]
Thanks for the reply but it's too late now, i have graduated now.

**Reply by onyedikachi23** [reply]
Alright 👌. Congratulations on your graduation 🎓.
